specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/DscpConfigurationDelete.json

```diff
@@ -7,7 +7,11 @@
   },
   "responses": {
     "200": {},
-    "202": {},
+    "202": {
+      "headers": {
```

Comment:
>headers [](http://example.com/codeflow?start=7&length=7)

Swagger document at [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/dscpConfiguration.json#L137C15-L137C16](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/dscpConfiguration.json#L137C15-L137C16)

do not show response headers. can you fix those as well, for all the updated examples ?